src/modules/reporting/components/dispute-market-card/dispute-market-card.jsx

```diff
@@ -61,6 +61,10 @@ DisputeMarketCard.propTypes = {
   history: PropTypes.object.isRequired,
   market: PropTypes.object.isRequired,
   isMobile: PropTypes.bool,
+  isForking: PropTypes.bool,
+  isForkingMarketFinalized: PropTypes.bool,
+  forkingMarket: PropTypes.string,
+  updateModal: PropTypes.func.isRequired,
 }
 
```

Comment:
changes are not needed MarketProperties will pick up auto-magically see comments above.